packages/zoe/src/contractFacet/types.js

```diff
@@ -81,7 +81,7 @@
  *
  * @param {ERef<Issuer>} issuerP Promise for issuer
  * @param {Keyword} keyword Keyword for added issuer
- * @returns {Promise<IssuerRecord>} Issuer is added and ready
+ * @returns {Promise<IssuerRecord<K>>} Issuer is added and ready
```

Comment:
`any` for keyword records since there's no way to know the actual type. We could keep `AssetKind` as the parameter, but that kind of means "all four of these" instead of "dunno"